### PR TITLE
boards: xtensa: esp32s2_franzininho: add support to the ESP32 S2 Franzininho educational development board.

### PR DESCRIPTION
**Pull request by uLipe**

### CODE REVIEW

**Comment by uLipe** [comment]
also @FBSeletronica, @joaodullius, PTAL, you might be interested on that.

File: boards/xtensa/esp32s2_franzininho/doc/index.rst
Comment (sylvioalves) [inline boards/xtensa/esp32s2_franzininho/doc/index.rst:9]: Perhaps `which is a highly integrated`

File: boards/xtensa/esp32s2_franzininho/doc/index.rst
Comment (sylvioalves) [inline boards/xtensa/esp32s2_franzininho/doc/index.rst:97]: This is no longer available and will be removed from documentation. It is required to download and manually install Espressif's custom openocd until Upstream version is available.

Comment (uLipe) [reply boards/xtensa/esp32s2_franzininho/doc/index.rst]: Removed debugging sections, this board does not have the debug connection after all.

**Review by sylvioalves — DISMISSED** [review]

**Review by LucasTambor — DISMISSED** [review]

**Comment by uLipe** [comment]
@cfriedt, @fabiobaltieri any chance of this gets merged in this release, we are organizing an Zephyr session based on this educational board.

**Comment by fabiobaltieri** [comment]
> @cfriedt, @fabiobaltieri any chance of this gets merged in this release, we are organizing an Zephyr session based on this educational board.

We are in stabilization phase so we'd have to ask TSC approval, will you show up at the TSC meeting if I flag the PR for discussion?

Two things beside that:
- can the second commit is just modifying/removing stuff added in the first one right? Can you squash the two? No sense keeping both in the history
- can you reduce the board image resolution? We are trying to keep the repository size under control and the photo is very detailed

**Review by LucasTambor — APPROVED** [review]

**Review by sylvioalves — APPROVED** [review]

**Review by glaubermaroto — APPROVED** [review]

**Comment by fabiobaltieri** [comment]
TSC decided to hold this for after the release. Updated the milestone tag accordingly.

**Comment by uLipe** [comment]
Hi folks, it seems now clear to merge, no? 

Thanks :)

**Review by cfriedt — APPROVED** [review]

**Pull request merged**
